torq.q

```diff
@@ -121,17 +152,19 @@ getconfig:{[path;level]
 
         /-get KDBCONFIG path
         conf:`$(kc:getenv[`KDBCONFIG]),"/",path;
-
+				
         /-if level is non-zero return appconfig and config files
         (),$[level;
           appconf,conf;
           first appconf,conf]}
 
+getdepfile:getconfig["config.csv";1]
```

Comment:
is getdepfile a function here, or a variable?  I think it is a variable but it is named like a function.  Can you change it?